src/content/1.7/development/page-reference/back-office/order/refunds/_index.md

```diff
@@ -0,0 +1,151 @@
+|| `bool $refundShippingCost`| Wether shipping cost should be refunded |
+|| `bool $generateCreditSlip`| Wether a credit splip should be generated |
+|| `bool $generateVoucher`| Wether a voucher should be generated |
+|| `int $voucherRefundType`| Id of the voucher refund type, those can be found in the file `VoucherRefundType.php` |
+| **IssueReturnProductCommand** |||
+|| `int $orderId` | The id of the order containing the products to be returned |
+|| `array $refunds`<br><br>Contains a list of refunds, each refund is an array containing an OrderDetail ID (int) and a quantity (int) to be refunded:<br><br>`[$orderDetailId => ['quantity' => $quantity]]`| An array containing a list of products (oderDetails) to be returned |
+|| `bool $restockRefundedProducts` | Wether returned product should be put back the stock (incrementing its quantity in stock) |
+|| `bool $refundShippingCost` | Wether shipping cost should be refunded |
+|| `bool $generateCreditSlip` | Wether a credit splip should be generated |
+|| `bool $generateVoucher` | Wether a voucher should be generated |
+|| `int $voucherRefundType`| Id of the voucher refund type, those can be found in the file `VoucherRefundType.php` |
+| **IssuePartialRefundCommand** |||
+|| `int $orderId`| The id of the order containing the products to be refunded |
+|| `array $refunds`<br><br>Contains a list of refunds, each refund is an array containing an OrderDetail ID (int), a quantity (int) and amount (float) to be refunded:<br><br>`[$orderDetailId => [`<br>`'quantity' => $quantity,'amount' => $amount]`<br>`]`| An array containing a list of products (oderDetails) to be refunded |
+|| `string $shippingCostRefundAmount` | Amount to be refunded for shipping cost |
+|| `bool $restockRefundedProducts` | Wether refunded products should be put back in the stock (incrementing its quantity in stock) |
+|| `bool $generateCreditSlip` | Wether a credit splip should be generated |
+|| `bool $generateVoucher` | Wether a voucher should be generated |
+|| `int $voucherRefundType`| Id of the voucher refund type, those can be found in the file `VoucherRefundType.php` |
+
+### The **actionProductCancel** hook
+
+Since prestashop 1.7.7, the `actionProductCancel` hook is triggered by all refund actions, it was not the case in 1.7.6, see the differences in the table below:
+
+|Does this action trigger the `actionProductCancel` hook ?| 1.7.6 | 1.7.7
+|:-------|:-------|:-----------------------|
+**Cancel product**| yes | yes |
+**Standard refund** | yes | yes |
+**Partial refund** | no | yes |
+**Return product** | yes | yes |
+
+#### How to know which action triggered the `actionProductCancel` hook ?
+
+##### In prestashop 1.7.7
+
+Since prestashop 1.7.7, the `actionProductCancel` hook has an `action` parameter indicating which action was performed. Its value is one of the attributes of the `CancellationActionType` class. 
```

Comment:
```suggestion
##### In Prestashop 1.7.7

Since Prestashop 1.7.7, the `actionProductCancel` hook has an `action` parameter indicating which action was performed. Its value is one of the attributes of the `CancellationActionType` class. 
```